VagrantProvision.sh

```diff
@@ -101,7 +106,7 @@ sudo /usr/bin/perl $HOOT_HOME/scripts/maven/SetMavenHttps.pl
 
 if ! grep --quiet "export HOOT_HOME" ~/.profile; then
     echo "Adding hoot home to profile..."
-    echo "export HOOT_HOME=~/hoot" >> ~/.profile
+    echo "export HOOT_HOME=$HOOT_HOME" >> ~/.profile
```

Comment:
Hard-coded paths are a no-no

Reply:
Hmmmm. I thought we changed that a while ago.